cdc-accelerator/feature/acl.js

```diff
@@ -0,0 +1,54 @@
+import path from 'path'
+import fs from 'fs'
+import { clearDirectoryContents } from '../utils/utils.js'
+import { SRC_DIRECTORY, BUILD_DIRECTORY } from './constants.js'
+import client from '../sap-cdc-toolkit/gigya/client.js'
+export default class ACL {
+    static ACL_FILE_NAME = 'Acls.json'
+    #credentials
+    constructor(credentials) {
+        this.#credentials = credentials
+    }
+
+    getName() {
+        return this.constructor.name
+    }
+
+    async init(aclID, partnerId, permissionGroupDirectory, dataCenter) {
+        let finalResponse = {}
+        for (const aclId of aclID) {
```

Comment:
Rename the var aclID to something that means a list, e.g aclIdList...
it's confusing with the other var called aclId